player-tester/src/chart.js

```diff
@@ -349,3 +348,19 @@ export function stopUpdatingChart() {
   clearTimeout(chartUpdatingIntervalId);
   chartIsUpdating = false;
 }
+
+
+export function exportChart(playerName) {
+  const data = {player: "rxplayer", data: config}
+  fetch(CDN_SERVER_URL+"/report", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    body: JSON.stringify(data),
+  }).then(() => {
+    window.alert(`Report enregistré`)
```

Comment:
Would be better without using alert :p

It moves the focus and prevent interaction with the page when we just want a feedback.